packages/core/src/attestation/Attestation.spec.ts

```diff
@@ -59,7 +63,7 @@ describe('Attestation', () => {
     )
   })
 
-  it('stores attestation', async () => {
+  xit('stores attestation', async () => {
```

Comment:
what's the issue here?

Reply:
I don't know, tbh. I will revert this and let's see